Client/Frontend/Widgets/GIFImage.swift

```diff
@@ -0,0 +1,181 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import SwiftUI
+
+struct GIFImage: UIViewRepresentable {
```

Comment:
Interesting, we do not use lottie for animations anymore?

Reply:
I wanted to mention this too but, it has already been designed as gif and implemented. We should use lottie animations, the gif images are holding **huge** space

Reply:
We decided to leave it as it is. Maybe something to improve later and for any future assets, gifs seem too filesize heavy for mobile apps